frontend/components/ticket/Ticket.vue

```diff
@@ -74,7 +74,7 @@
                 {{ status[ticket.status] }}
               </div>
               <div class="mt-2 flex items-center text-sm leading-5 text-gray-500" title="Assigned to"
-                   v-if="ticket.assignee">
+                   v-if="ticket.assignee.username">
```

Comment:
Is het hier niet handiger om ticket.assignee && ticket.assignee.username te doen?